src/vcpkg-test/util.cpp

```diff
@@ -109,6 +109,15 @@ namespace vcpkg::Test
 
     static void check_json_eq(const Json::Value& l, const Json::Value& r, std::string& path, bool ordered);
 
+    static void double_set_difference(const std::set<std::string>& a,
```

Comment:
```suggestion
    static void symmetric_difference(const std::set<std::string>& a,
```

Reply:
Symmetric difference typically refers to the union of the two results of this function. This function instead reports them separately and so should have a different name.